### PR TITLE
DEBUG - Run testMaxLayoutGrowth in TF

### PR DESCRIPTION
**Pull request by mvollmer**
May it hang!

### CODE REVIEW

File: test/verify/check-storage-lvm2 [hunk 1]
```diff
@@ -666,7 +666,12 @@ class TestStorageLvm2(storagelib.StorageCase):
             width = b.call_js_func('(function (sel) { return ph_find(sel).offsetWidth; })', slider)
             b.mouse(slider, "click", width, 0)
             self.dialog_apply()
-            self.dialog_wait_close()
+            try:
+                self.dialog_wait_close()  # wait for grow to finish
+            except testlib.Error:
+                # Produce evidence of lvm2 hanging, so that our maughties can match on it
+                print(m.execute("for p in $(pgrep lvm); do cat /proc/$p/stack; done"))
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply test/verify/check-storage-lvm2:673]: Yes, exactly.

**Comment by martinpitt** [comment]
Not sure if it helps: in #20046 I saw an [arch run](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20046-20240222-090533-751c42ed-arch-storage/log.html#38-2) where TestStorageLvm2.testMaxLayoutSizes fails as well, and the [raw log](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20046-20240222-090533-751c42ed-arch-storage/log) shows that there's a follow-up failure (TestStorageStratis.testBasic) which looks like VM corruption.

**Comment by mvollmer** [comment]
The first round was all green. :-(

**Comment by martinpitt** [comment]
Some examples from today, in case it helps with collecting journals:
 - f40: https://artifacts.dev.testing-farm.io/99b9d6c1-3dbb-4d18-bd2a-ac7baf47e9a9/
 - f40: https://artifacts.dev.testing-farm.io/087148ad-39b8-45cf-bc72-c58e70f13e73/
 - f39: https://artifacts.dev.testing-farm.io/911d693b-930c-4cce-912f-ddf5e26f1076/

I was quite surprised to see this on F39 too, but Fedora aggressively updates kernels in stable releases, too.

**Comment by mvollmer** [comment]
> * f39: https://artifacts.dev.testing-farm.io/911d693b-930c-4cce-912f-ddf5e26f1076/

This look like a too short timeout.

I can't get the MaxLayoutGrowth to hang here, maybe that was also just a too short timeout...

**Comment by mvollmer** [comment]
Second round is also all green.

**Comment by mvollmer** [comment]
And the third is green. Must have been a different day yesterday for TF. Triggering was slow as well. So I give up with this.

**Comment by martinpitt** [comment]
Another instance: https://cockpit-logs.us-east-1.linodeobjects.com/pull-20130-20240301-123636-65d46d58-arch-storage/log.html#39-2

**Pull request closed without merging**